commit ca811a7f8338f24001ad5f7cebfa21dbfccd5702
Author: John Johansen <john.johansen@canonical.com>
Date:   Thu Sep 28 13:47:13 2017 -0400

    UBUNTU: SAUCE: LSM stacking: make sure LSM blob align on 64 bit boundaries
    
    The security->task blob reserving the first 12 bytes means that LSM
    blobs don't align on 64 byte boundaries. This is not a problem
    for x86 but if an LSM stores a long or ptr in its blob, then some
    architectures require it be aligned to the arch word size and
    will through a fault.
    
    Signed-off-by: John Johansen <john.johansen@canonical.com>
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/security/security.c b/security/security.c
index 9be710e9ca64..57c1383b35a2 100644
--- a/security/security.c
+++ b/security/security.c
@@ -51,7 +51,7 @@ char *lsm_names;
  */
 static struct lsm_blob_sizes blob_sizes = {
 #ifdef CONFIG_SECURITY_STACKING
-	.lbs_task = SECURITY_NAME_MAX + 2,
+	.lbs_task = SECURITY_NAME_MAX + 6,
 #endif
 };
 
